modules/bluima_opennlp/src/main/java/ch/epfl/bbp/shaded/opennlp/tools/lang/english/SentenceDetector.java

```diff
@@ -40,11 +41,26 @@
    * Loads a new sentence detector using the model specified by the model name.
    * @param modelName The name of the maxent model trained for sentence detection. 
    * @throws IOException If the model specified can not be read.
+   *
+   * TODO remove me (?)
```

Comment:
Sure. This constructor is only used in the main function below so if it makes sense I can remove it completely and use `ModelProxy` to load the model. I just don't know if this program is used somewhere else -- in that case it will not work as before.